tests/input_events_test_basic.expected

```diff
@@ -1,4 +1,3 @@
 Checker driving port
 Checker expecting 0x\w as trigger data
 input_gpio task does not support events.
-Unhandled exception: ECALL, data: 0x00000000
```

Comment:
I just want to understand this change.  Are we saying that Tools 15.1.4 alters the run-time behaviour of the executable so that it no longer issues a run-time exception?  When built with the old tools, why did the execpetion occur?  What change in Tools 15.1.4 to keep the exeception frm happening?

Reply:
To fix the issue in the description the unhandled exceptions are not stored in a text file when using xmostest, the change in the code of xsim is here:
https://github0.xmos.com/xmos-int/xsim/pull/66 
This change was needed for lib_xud and Ross confirmed it is necessary.

Reply:
From my understanding we could change the code in tools_xmostest, but I don't think it is worth the effort. I would like to point out that the line "input_gpio task does not support events." in the expected output is issued by the assert in the code, so the test coverage is not compromised by this PR>

Reply:
Okay, thanks for this information.